gallery-so/gallery
# [GAL-3961] Curated Feed mobile

### Summary of Changes

1. Changed from `trendingFeed` to `curatedFeed`
2. Rename all treding to curated

### Demo or Before and After


https://github.com/gallery-so/gallery/assets/4480258/3c912bd9-79b0-4eb0-94dc-7b0bf42e1624


### Edge Cases

- Tested the curated feed without any warning
- The same post/event display more than once in feed

### Testing Steps

1. Login in the app
2. You should see there is a **Curated** tab and click it to browse the feed

### Checklist

Please make sure to review and check all of the following:
